apps/electron/package.json

```diff
@@ -67,6 +67,7 @@
     "lodash-es": "^4.17.21",
     "nanoid": "^4.0.2",
     "rxjs": "^7.8.1",
+    "which": "^3.0.1",
```

Comment:
why you need this?

Reply:
I added this in order for spawn to find yarn. I've added a screenshot of the error I'm getting in the description of the pr.

Reply:
this package should in dev deps